c2corg_ui/static/js/card.js

```diff
@@ -107,4 +107,94 @@ app.CardController.prototype.createURL = function() {
 };
 
 
+/**
+ * Gets the global ratings for each activity of a route.
+ * @export
+ * @return {Object} rating
+ */
+app.CardController.prototype.getGlobalRatings = function() {
+  var doc = this.doc;
+  var rating = {};
+
+  for (var i = 0; i < doc.activities.length; i++) {
+    var a = doc.activities[i];
+    if (a === 'rock_climbing' || a === 'mountain_climbing' || a === 'snow_ice_mixed') {
```

Comment:
I don't understand what you mean. What's the problem of the following code? It's a simple switch on the value of a variable.
```
switch (activitiy) {
  case 'rock_climbing':
  case 'mountain_climbing':
  case 'snow_ice_mixed':
    rating['global_rating'] = doc.global_rating;
    break;
  case 'via_ferrata':
    rating['via_ferrata_rating'] = doc.via_ferrata_rating;
    break;
  case 'snowshoeing':
    ...
}
```